crypto/ledger_mock.go

```diff
@@ -8,15 +8,15 @@ import (
 	"github.com/btcsuite/btcd/btcec"
 	"github.com/pkg/errors"
 
+	secp256k1 "github.com/tendermint/btcd/btcec"
+	"github.com/tendermint/tendermint/crypto"
+	tmsecp256k1 "github.com/tendermint/tendermint/crypto/secp256k1"
+
 	bip39 "github.com/cosmos/go-bip39"
 
 	"github.com/cosmos/cosmos-sdk/crypto/keys/hd"
 	"github.com/cosmos/cosmos-sdk/tests"
-	"github.com/cosmos/cosmos-sdk/types"
-
-	secp256k1 "github.com/tendermint/btcd/btcec"
-	"github.com/tendermint/tendermint/crypto"
-	tmsecp256k1 "github.com/tendermint/tendermint/crypto/secp256k1"
+	sdk "github.com/cosmos/cosmos-sdk/types"
```

Comment:
I like the general concept/rule behind: 
`authtypes "github.com/cosmos/cosmos-sdk/x/auth/types`

for consistency, should this be `types` or worst case `sdktypes`.
`sdk` is not very clear.

Reply:
I'm actually a fan of calling this `sdk`. Most of the core functionality and common types are in that package.

Reply:
I agree with fede, `sdktypes` is too verbose, + it's used everywhere, I think `sdk` is optimum